docs/pages/index.mdx

```diff
@@ -6,102 +6,81 @@ layout: tocless-doc
+- Gather audit log events and session recordings.
+- Automatically onboard and offboard users with integrations such as Okta, GitHub, and Google Workspace.
 
 ## Try out Teleport
 
-Quickly see how Teleport works in one of our demo environments.
+- [Teleport Cloud](https://goteleport.com/signup/): Deploy a free cloud trial.
+- [Linux Server](./try-out-teleport/linux-server.mdx): Deploy on a Linux Server.
+- [Docker Compose](./try-out-teleport/docker-compose.mdx): Deploy locally with Docker Compose.
+- [Kubernetes](./try-out-teleport/local-kubernetes.mdx): Deploy locally with Kubernetes.
+- [DigitalOcean](./try-out-teleport/digitalocean.mdx): Deploy with our 1-Click DigitalOcean droplet.
 
-- [Linux Server](./try-out-teleport/linux-server.mdx): Deploy Open Source
-  Teleport on a Linux Server.
-- [Digital Ocean](./try-out-teleport/digitalocean.mdx): Use our 1-Click Digital
-  Ocean droplet to set up Teleport quickly.
-- [Browser Labs](https://play.instruqt.com/teleport/invite/imz8g1n1xzru): Try Teleport using our guided
-  interactive labs.
-- [Docker Compose Lab](./try-out-teleport/docker-compose.mdx): Try Teleport
-  locally using Docker Compose.
-- [Kubernetes Lab](./try-out-teleport/local-kubernetes.mdx): See how Teleport
-  runs on Kubernetes with this local lab.
+Want more details on the [different editions of Teleport](./choose-an-edition/introduction.mdx)?
 
-## Choose an edition
+## Connect to Teleport
 
-Read one of our Getting Started guides to see how to deploy open source
-Teleport, Teleport Enterprise, or Teleport Cloud.
-
-You can also [compare Teleport editions](./choose-an-edition/introduction.mdx).
-
-- [Open Source Teleport](./try-out-teleport/linux-server.mdx): Learn how to host your own open source Teleport deployment on a standalone Linux server.
-- [Teleport Enterprise](./choose-an-edition/teleport-enterprise/introduction.mdx): Get started with a self-hosted Teleport Enterprise deployment, which gives you more advanced features and full customization.
-- [Teleport Cloud](./choose-an-edition/teleport-cloud/getting-started.mdx): Try our cloud-hosted version for free.
-
-## Configure access
-
-Secure your infrastructure while keeping your engineers productive.
-
-<ScopedBlock scope={["cloud", "enterprise"]}>
-- [Set up SSO](./access-controls/sso.mdx): Configure Teleport's integration with your SSO provider so you can automatically on– and off-board users.
-</ScopedBlock>
-<ScopedBlock scope={["oss"]}>
-- [Set up SSO](./access-controls/sso/github-sso.mdx): Configure Teleport's integration with GitHub so you can automatically on– and off-board users.
-</ScopedBlock>
-- [Define roles](./access-controls/guides/role-templates.mdx): Manage who can access which parts of your infrastructure.
-
-## Add your infrastructure
-
-Use Teleport to provide secure access to all of your infrastructure.
+Already have Teleport running? Start connecting your infrastructure to provide secure access.
```

Comment:
Would a user go right from from trying out Teleport to connecting resources? I suppose they could, but I think it would make sense to first direct the reader to our pages on deploying a production cluster or setting up a Teleport Enterprise Cloud account.